apple/internal/partials/debug_symbols.bzl

```diff
@@ -202,6 +273,10 @@ def _debug_symbols_partial_impl(
         if _AppleDebugInfo in x
     ]
 
+    debug_output_filename = bundle_name
```

Comment:
@thii This line changes the dSYM filename when a custom executable name is used.
Shouldn't we use the `executable_name` instead of the `bundle_name`?
(According to your PR: https://github.com/bazelbuild/rules_apple/pull/934)

Reply:
Yes, it should be `executable_name` if it's not `None` (I think there's a helper from `bundling_support`). Are you able to submit a fix?

Reply:
I see. I'll test with a patch and file a PR if it works.